prime-dotnet-webapi/Models/SiteRegistration/ExceptionalRemoteAccessSite.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Prime.Models
+{
+    [Table("ExceptionalRemoteAccessSite")]
+    public class ExceptionalRemoteAccessSite
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public string PEC { get; set; }
+
+        public string RegistrationId { get; set; }
```

Comment:
I think this needs a comment ... "RegistrationId of Organization that Site is associated with, for additional verification"?

Reply:
add comment to the class and the registration ID. Hope that clarify the purpose of the class and its property.